gallerypage.py

```
@@ -1144,36 +1042,73 @@ def _write_jekyll(self, filepath, remote_keys, replacements,
-   
-            f.write('</table>\n')
-            f.write('<br>\n')
+                    row = {
+                        'table_row': name,
+                        'value': value[0].encode('ascii', 'xmlcharrefreplace').decode('ascii'),
+                        'additional_values': value[1].encode('ascii', 'xmlcharrefreplace').decode('ascii')
+                    }
+
+                yaml_data['catalog_table'].append(row)
+
+            yaml_output = yaml.dump(yaml_data, Dumper=galleries.FixIndent, default_flow_style=False, sort_keys=False)
```

Ruff (0.15.2)

[error] Undefined name `galleries`

(F821)